Subscription event extented

don-aot opened the pull request:
<!-- Provide a general summary of your changes in the Title above -->
# Description

<!-- Describe your changes in detail -->
<!-- Why is this change required? What problem does it solve? -->
<!-- If it fixes an open issue, please link to the issue here. -->

## Types of changes

<!-- What types of changes does your code introduce? Uncomment all that apply: -->

<!-- Bug fix (non-breaking change which fixes an issue) -->
<!-- New feature (non-breaking change which adds functionality) -->
<!-- Documentation (non-breaking change with enhancements to documentation) -->
<!-- Breaking change (fix or feature that would cause existing functionality to change) -->

## Checklist

<!-- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!-- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->

- [ ] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [ ] I have checked that unit tests pass locally with my changes
- [ ] I have run the npm script lint on the frontend and backend
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] I have approval from the product owner for the contribution in this pull request

## Further comments

<!-- If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc... -->

don-aot committed:
8349:CHEFS Front End

CRED for the subscription settings of a form

don-aot committed:
8349:CHEFS Front End

Removing old code

jimmyPasta commented on app/src/db/migrations/20230614063945_031-form-subscribe.js:
```diff
@@ -17,11 +15,11 @@ exports.up = function (knex) {
           inner join form f on fv."formId" = f.id
-          inner join form_subscription fs on s."formVersionId" = fs."formVersionId"
+          inner join form_subscription fs on f.id = fs."formVersionId"
       `)
```
This can not have the f.id = fs.formVersionId

don-aot committed:
8349:CHEFS Front End

Translation Changes

don-aot committed:
8349:CHEFS Front End

Unit Testing

don-aot committed:
8349:CHEFS Front End

Code review fixes

don-aot committed:
8349:CHEFS Front End

Adding an authorization key to the configurations and removing the newly created view for subscription settings

don-aot committed:
8349:CHEFS Front End

Validation Updates
